### DISCUSSION TITLE
While add UI Bootstrap Switch plugin, Get uncaught exception

### DISCUSSION THREAD

**Discussion by ilksenbozok** [serenity-is/Serenity#6783] [Help] [answered]
Hi everyone,
I was trying to use bootstrap switch by following the Bootstrap Switch plugin [post](https://github.com/serenity-is/Serenity/wiki/UI-Bootstrap-Switch-plugin-(by-marcobisio)). Although I did everything as stated, I got an error like the following in the Console.

![image](https://github.com/serenity-is/Serenity/assets/27086236/5025d690-fff5-44ba-a863-4d5b05c28c7e)

Here is my Bootstrap switch widget : 
![image](https://github.com/serenity-is/Serenity/assets/27086236/d8f32372-f730-4edb-b591-dae66d59fc92)

how can i fix this ? Thanks in advance.

**Comment by wezmag** [comment, marked as answer]
The second part of the error message tells you how to fix the issue. You need to add `@Decorator.registerEditor('MyProject.MyModule.MyEditor')`. Please change MyProject.MyModule.MyEditor accordingly.

BTW, you don't need to use namespace now.

**Reply by ilksenbozok** [reply]
Thanks for the reply. It helped me a lot :)
